Frontend/frontend/.env

```diff
@@ -1 +1 @@
-REACT_APP_BACKEND_API="https://localhost:44394/"
\ No newline at end of file
+REACT_APP_BACKEND_API="https://localhost:5001/"
```

Comment:
You should unhardcode it. Maybe set it as an environment variable so other people who run can with different local hosts can change it easily.

Reply:
This line was not added by him, he just changed the number, probably automatically when running it. It was still hard coded to a different number before hand, and so changing how it works is out of scope of this PR